lib/fizzy/parser.cpp

```diff
@@ -489,6 +489,8 @@ Module parse(bytes_view input)
         switch (import.kind)
         {
         case ExternalKind::Function:
+            if (import.desc.function_type_index >= module.typesec.size())
+                throw validation_error{"unknown type"};
```

Comment:
```suggestion
                throw validation_error{"invalid type index of an imported function"};
```

Reply:
This was a string from spec tests, but I agree more specific message is better